### PR TITLE
Maintain class names through build

### PR DESCRIPTION
**Pull request by vbarzokas**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,11 +1,23 @@
...
+import { JssProvider } from 'react-jss';
+import { createGenerateClassName } from '@material-ui/core/styles';
 import './index.css';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+const generateClassName = createGenerateClassName({
+  dangerouslyUseGlobalCSS: true, // won't minify CSS classnames when true
+  productionPrefix: 'mow', // 'jss' by default
```
Comment (KamuelaFranco) [inline src/index.js]: Do we need this at all?

File: package.json [hunk 2]
```diff
@@ -46,6 +47,7 @@
     "eslint-plugin-import": "^2.20.1",
     "eslint-plugin-jsx-a11y": "^6.2.3",
     "eslint-plugin-react": "^7.18.3",
-    "eslint-plugin-react-hooks": "^1.7.0"
+    "eslint-plugin-react-hooks": "^1.7.0",
+    "serve": "^11.3.0"
```
Comment (msk-) [inline package.json]: Are we installing this incidentally? I guess it's handy for development..

Comment (KamuelaFranco) [reply package.json]: Let's not though.

Comment (KamuelaFranco) [inline package.json]:
```suggestion
```

File: src/index.js (hunk 1 above)
Comment (KamuelaFranco) [inline src/index.js]:
```suggestion
```

**Pull request merged**